esmvaltool/reformat.py

```diff
@@ -0,0 +1,45 @@
+"""Port reformat.py from v1 to v2 (life hack)."""
+import fnmatch
+import os
+from subprocess import call
+
+
+def cmor_reformat(config, logvar):
+    """Run the oldskool v1 reformat scripts."""
+    logvar.info("Running the oldschool reformatting scripts.")
+
+    # set the reformat scripts dir
+    reformat_scripts = os.path.join(os.path.dirname(__file__),
+                                    'reformat_scripts')
+
+    # assemble i/o information
+    project_info = {}
+    raw_obs = config["rootpath"]["RAWOBS"][0]
+    for _, datasets, _ in os.walk(raw_obs, followlinks=True):
```

Comment:
We should not loop over all datasets in RAWOBS, but just call the one specified as command line argument (e.g., DUMMY).

Reply:
@mattiarighi there is no such command line option as DUMMY, DUMMY replaces recipe, I thought the whole point was to batch reformat a whole lot of datasets in RAWOBS

Reply:
See [my comment above](https://github.com/ESMValGroup/ESMValTool/pull/666#issuecomment-430669906). The idea is to pass the dataset name as an argument. So if I pass "DUMMY", the tool should call `cmorize_obs_DUMMY`.

In any case, we do not want to process all datasets in RAWOBS.

Reply:
ok cool, will get that done, wouldn't this be crazy since if you want to cmorize 20 models you will have to run 20 times? anyway, we can probably make it read the datasets to cmorize from a list

Reply:
Yes, but going through all datasets in RAWOBS is even crazier.
A list would be the compromise solution. :handshake: